apps/web/src/middleware.ts

```diff
@@ -37,14 +42,19 @@ export default authMiddleware({
       const result = await db
         .select()
         .from(usersToWorkspaces)
-        .leftJoin(userQuery, eq(userQuery.id, usersToWorkspaces.userId))
+        .innerJoin(userQuery, eq(userQuery.id, usersToWorkspaces.userId))
         .execute();
+
       if (result.length) {
         const orgSelection = new URL(
           `/app/${result[0].users_to_workspaces.workspaceId}`,
           req.url,
         );
         return NextResponse.redirect(orgSelection);
+      } else {
+        // return NextResponse.redirect(new URL("/app/onboarding", req.url));
```

Comment:
@mxkaske that's where we should probably do something when the webhook is not received yet. 

redirect to a small onboarding page?

Reply:
Let's see what we can do. I don't mind having a Loading screen with a small description what is happening. Being open about the status.